engine/src/test/java/com/datatorrent/stram/client/AppPackageTest.java

```diff
@@ -57,24 +55,17 @@
   String appPackageDir = "src/test/resources/testAppPackage/mydtapp";
 
   @BeforeClass
-  public static void starting()
+  public static void starting() throws IOException, JSONException
   {
+    File file = StramTestSupport.createAppPackageFile();
```

Comment:
if an exception is thrown before application package is created, there is no need to close it.